### PR TITLE
Add some tooltips to the program popup

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

**Comment by cquiroz** [comment]
image

File: explore/src/main/scala/explore/EditableLabel.scala [hunk 1]
```diff
@@ -59,29 +69,72 @@ object EditableLabel {
...
+      editButtonTooltip,
       deleteButtonClass,
-      deleteButtonIcon
+      deleteButtonIcon,
+      deleteButtonTooltip,
+      okButtonTooltip,
+      discardButtonTooltip
     )
 
+  type Editing = Editing.Type
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/EditableLabel.scala:80]: Can this just be an `enum Editing: case NotEditing, InEdition` ?

**Review by rpiaggio — APPROVED** [review]
LGTM

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/EditableLabel.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/EditableLabel.scala:80]: Wouldn't be cheaper to use an opaque type?

Comment (rpiaggio) [reply explore/src/main/scala/explore/EditableLabel.scala:80]: Generally speaking, yes. If we define `val`s for the values, probably yes too, but no so much ;) Ohh unless the vals are inline?

**Pull request merged**